<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Расписание автобусов</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="locale">
   <locale language="Russian" country="Belarus"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>&amp;Помощь</string>
    </property>
    <addaction name="action_about_program"/>
    <addaction name="action_about_Qt"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_back"/>
   <addaction name="action_forward"/>
   <addaction name="separator"/>
   <addaction name="action_check_for_updates"/>
  </widget>
  <action name="action_back">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="Resources/resources.qrc">
     <normaloff>:/icons/back</normaloff>:/icons/back</iconset>
   </property>
   <property name="text">
    <string>&amp;Назад</string>
   </property>
   <property name="toolTip">
    <string>Назад</string>
   </property>
   <property name="shortcut">
    <string>Alt+Left</string>
   </property>
  </action>
  <action name="action_forward">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="Resources/resources.qrc">
     <normaloff>:/icons/forward</normaloff>:/icons/forward</iconset>
   </property>
   <property name="text">
    <string>&amp;Вперёд</string>
   </property>
   <property name="toolTip">
    <string>Вперёд</string>
   </property>
   <property name="shortcut">
    <string>Alt+Right</string>
   </property>
  </action>
  <action name="action_about_program">
   <property name="text">
    <string>О &amp;программе</string>
   </property>
  </action>
  <action name="action_about_Qt">
   <property name="text">
    <string>&amp;О Qt</string>
   </property>
  </action>
  <action name="action_check_for_updates">
   <property name="icon">
    <iconset resource="Resources/resources.qrc">
     <normaloff>:/icons/checkForUpdates</normaloff>:/icons/checkForUpdates</iconset>
   </property>
   <property name="text">
    <string>Проверить наличие обновлений</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
